### DISCUSSION TITLE
Email error

### DISCUSSION THREAD

**Discussion by Product-UX** [RedHat-UX/red-hat-design-system#1702] [Design library feedback]
Hello,
I have tried to send an Email to design-system@redhat.com to request a redhat email to login into Figma but I receive an error message. Is there another way to get access to the Figma RHDS library file.
Thanks

**Comment by bennypowers** [comment]
@markcaron @coreyvickery can you take a look at this next week?

**Reply by bennypowers** [reply]
Thanks for bringing this up @Product-UX it appears to be solved on our end.

**Reply by coreyvickery** [reply]
@Product-UX Are you an employee of Red Hat?
